source/_integrations/gentex_homelink.markdown

```diff
@@ -0,0 +1,52 @@
+---
+title: HomeLink
+description: Instructions on how to setup your HomeLink enabled car with Home Assistant.
+ha_category:
+  - Binary sensor
+ha_release: 2025.1
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@niaexa'
+  - '@ryanjones-gentex'
+ha_domain: gentex_homelink
+ha_platforms:
+  - binary_sensor
+ha_integration_type: integration
+ha_quality_scale: bronze
+---
+
+[HomeLink by Gentex](https://www.gentex.com/products-technology/automotive/connected-car/) is the most widely used and trusted car-to-home automation system, with nearly 100 million HomeLink-equipped vehicles on the road today.
+
+HomeLink consists of vehicle-integrated programmable buttons that use radio frequency to conveniently and reliably operate your garage door opener, security gate and other radio frequency-compatible devices.
+
+This integration provides the following platform interfaces:
+
+- Binary sensors: the system detects when the programmable buttons are pressed. Users can use these presses to trigger home automation routines.
+
+## The HomeLink® App
```

Comment:
Please restructure the HomeLink App section and the Configuration section into a list of prerequisites, and a configuration section. 
you can find an example documentation structure on the [examples page], and in the [developer documentation](https://developers.home-assistant.io/docs/documenting/integration-docs-examples)

For the configuration section, you can use this [standard config_flow text block here](https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L51). See also https://developers.home-assistant.io/docs/documenting/integration-docs-examples#configuration